plugins/woocommerce/patterns/coming-soon-entire-site.php

```diff
@@ -38,14 +38,8 @@
 <!-- /wp:group -->
 
 <!-- wp:group {"style":{"spacing":{"blockGap":"48px"}},"className":"woocommerce-coming-soon-social-login","layout":{"type":"flex","flexWrap":"nowrap"}} -->
-<div class="wp-block-group woocommerce-coming-soon-social-login"><!-- wp:social-links {"iconColor":"contrast","iconColorValue":"#111111","style":{"layout":{"selfStretch":"fit","flexSize":null},"spacing":{"blockGap":{"left":"18px"}}},"className":"is-style-logos-only"} -->
-<ul class="wp-block-social-links has-icon-color is-style-logos-only"><!-- wp:social-link {"url":"https://www.linkedin.com/","service":"linkedin"} /-->
-
-<!-- wp:social-link {"url":"https://www.instagram.com","service":"instagram"} /-->
-
-<!-- wp:social-link {"url":"https://www.facebook.com","service":"facebook"} /--></ul>
-<!-- /wp:social-links -->
-
+<div class="wp-block-group woocommerce-coming-soon-social-login">
```

Comment:
Do we need the div to be here? Could it be inside `plugins/woocommerce/templates/parts/coming-soon-social-links.html` instead?

Reply:
We need the div to be here because it's a wrapper for the social links and **login button**. If we can put the login button inside the social links template part, then we can move the div to the social links template part. However, not all our coming soon templates have a login button, so I didn't use that.

Reply:
I see, that makes sense. Thanks!